test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompactionTestUtils.java

```diff
@@ -144,11 +146,11 @@ public static void createTable(AccumuloClient client, String tableName, String s
 
   }
 
-  public static void writeData(AccumuloClient client, String table1)
+  public static void writeData(AccumuloClient client, String table1, int rows)
       throws MutationsRejectedException, TableNotFoundException, AccumuloException,
```

Comment:
```suggestion
      throws TableNotFoundException, AccumuloException,
```
Looks like this is covered under AccumuloException